Core/Core/Credentials/AccountManager.cs

```diff
@@ -705,30 +703,42 @@ await response.Content.ReadAsStringAsync().ConfigureAwait(false)
   }
 
   /// <summary>
-  /// Check the <paramref name="headers"/> for a <c>"x-speckle-frontend-2"</c> <see cref="Boolean"/> value
+  /// Check the <paramref name="server"/> for having a <c>"x-speckle-frontend-2"</c> <see cref="Boolean"/> header value
```

Comment:
Maybe we can re-word this summary also. We aren't checking the server url.. we're making a request to the server. Same goes for the `returns` and `exception` tag

Reply:
I agree, it should be very clear what is the behaviour of this.